iroh/src/node/rpc/docs.rs

```diff
@@ -249,9 +246,9 @@ impl DocsEngine {
     pub fn doc_get_many(
         &self,
         req: DocGetManyRequest,
-    ) -> impl Stream<Item = RpcResult<DocGetManyResponse>> {
+        tx: flume::Sender<anyhow::Result<SignedEntry>>,
+    ) {
         let DocGetManyRequest { doc_id, query } = req;
-        let (tx, rx) = flume::bounded(ITER_CHANNEL_CAP);
```

Comment:
Ah, you already had the flume channel and just moved it outside?

It seems that the signature of the quic-rpc handler is not ideal, or at least not always.

Maybe it would be better if instead of returning a stream you would get a sink. Would that help?

Reply:
Switched to using `Either`